src/renderActivitie.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import activities from './activities.js';
+
+export const activCtn = document.querySelector('#activitiesContainer');
+
+export const renderFunction = (JsonObjectsArray) => {
+  activCtn.innerHTML = '';
+  if (JsonObjectsArray === null || JsonObjectsArray.length === 0) {
+    return;
+  }
+  JsonObjectsArray.forEach((act) => {
+    const actElmnt = document.createElement('div');
+    actElmnt.classList.add('px-3');
+    actElmnt.classList.add('element');
+    actElmnt.classList.add('d-flex');
+    actElmnt.classList.add('border');
+    actElmnt.classList.add('align-items-center');
+    actElmnt.classList.add('justify-content-between');
+    actElmnt.draggable = true;
+
+    const chkBx = document.createElement('input');
+    chkBx.type = 'checkbox';
+    chkBx.id = _.join(['ck', act.index], '');
+    chkBx.checked = act.completed;
+    actElmnt.appendChild(chkBx);
+
+    const chkLabel = document.createElement('input');
+    chkLabel.value = act.description;
+    chkLabel.classList.add('w-100');
+    chkLabel.classList.add('px-2');
+    chkLabel.classList.add('border-0');
+    chkLabel.id = _.join(['lb', act.index], '');
+    chkLabel.htmlFor = act.index;
+    actElmnt.appendChild(chkLabel);
+
+    const myImg = document.createElement('img');
+    myImg.classList.add('verticalDots');
+    myImg.id = _.join(['dl', act.index], '');
+    actElmnt.appendChild(myImg);
+
+    activCtn.appendChild(actElmnt);
+
+    chkBx.addEventListener('change', (e) => {
```

Comment:
In this case, I would only suggest that you add some comments to section the functions that are acting to have a better reading of the code